db/schema.rb

```diff
@@ -11,7 +11,7 @@
 #
 # It's strongly recommended to check this file into your version control system.
 
-ActiveRecord::Schema.define(:version => 20170202190333) do
+ActiveRecord::Schema.define(:version => 20170410201106) do
```

Comment:
When resolving the conflict you should make sure the schema references the larger number.